site/topics/memory/latches-flipflops.rst

```diff
@@ -185,6 +185,49 @@ S-R Latch with Enable
 Data Latch (D Latch)
 ====================
 
+* The S-R latch can be modified to eliminate the ability for both set and reset to be high at the same time
+
+    * Make it such that only one can be active at a time
+
+
+* This can be achieved by having a single input split into two signals
+
+    * One of the two parts of the split signal is fed into the S-R latch
+    * The other part is inverted before being fed into the S-R latch
+
+
+.. figure:: D_latch.png
+    :width: 500 px
+    :align: center
+
+    A D latch, which is a modified S-R latch. Here, :math:`D` is the label for the input data. Like the S-R latch,
+    :math:`Q` is the bit being stored, and :math:`\lnot Q` is the inverse of the bit being stored. The labels
+    :math:`S` and :math:`R` are included here as a reference to what these inputs were called in the S-R latch, but
+    are not typically included in diagrams of D latches.
+
+
+* The above figure shows how the S-R latch may be modified into what is called a Data latch (D latch)
+
+    * The :math:`D` means data
```

Comment:
I legit had no idea. I figured it was just a name. D.

Reply:
Your comment worried me enough to verify that it was not an assumption I had. Wiki at least corroborates.